Warning in SelfDestructParticleSystem.cs

```
Assets/Plugins/CandyCoded/Scripts/Components/SelfDestructParticleSystem.cs(10,36): warning CS0109: The member `CandyCoded.SelfDestructParticleSystem.particleSystem' does not hide an inherited member. The new keyword is not required
```

## Specifications

  - Version of CandyCoded: v1.0.0-beta.2
  - Operating System: MacOS 10.13.4 Beta
  - Unity: 2017.4.0f1

--- a/Assets/Plugins/CandyCoded/Scripts/Components/SelfDestructParticleSystem.cs
+++ b/Assets/Plugins/CandyCoded/Scripts/Components/SelfDestructParticleSystem.cs
@@ -7,15 +7,15 @@ namespace CandyCoded
     {
 
         [SerializeField]
-        private new ParticleSystem particleSystem;
+        private ParticleSystem ps;
 
         private void Awake()
         {
 
-            if (particleSystem == null)
+            if (ps == null)
             {
 
-                particleSystem = gameObject.GetComponent<ParticleSystem>();
+                ps = gameObject.GetComponent<ParticleSystem>();
 
             }
 
@@ -24,7 +24,7 @@ namespace CandyCoded
         private void LateUpdate()
         {
 
-            if (particleSystem && !particleSystem.IsAlive())
+            if (ps && !ps.IsAlive())
             {
 
                 Destroy(gameObject);

